dashboard-widgets.el

```diff
@@ -85,13 +85,22 @@ If the value is nil then no banner is displayed.")
                                      (agenda    . dashboard-insert-agenda)
                                      (registers . dashboard-insert-registers)))
 
-(defvar dashboard-items '((recents   . 5)
-                          (bookmarks . 5)
-                          (agenda    . 5))
-  "Association list of items to show in the startup buffer.
-Will be of the form `(list-type . list-size)`.
-If nil it is disabled.  Possible values for list-type are:
-`recents' `bookmarks' `projects' `agenda' `registers'")
+(defcustom dashboard-items '((recents   . 5)
+                             (bookmarks . 5)
+                             (agenda    . 5))
+  "List of sections and items to show in the startup buffer."
+  :type '(repeat
```

Comment:
What happens here with custom widgets? If I'm understanding this `:type` section, it defines that that the items in the list can only be one of the first default widgets types, right?

Reply:
Yes, can you give me an example of a custom widget so I can update the type?

Reply:
I have an example in my own configuration: https://gitlab.com/JesusMtnez/emacs.d/blob/master/core/20-core-ui.el#L26

But everyone should be able to define their own widgets if they want.